### ISSUE TITLE
SSO + change user button

### ISSUE THREAD

**Issue by JonasKs** [jobec/django-auth-adfs#64]
The idea of disabling SSO is great and its no longer needed for us to open Firefox in incognito mode if we want to use another user. What would be the best way of integrating the SSO solution, but with the option to change the user? I'd rather not do it static and dirty as I first started doing 😄 

`Set an unusable password on newly created user instead of leaving it empty.` also enforces all users to be deleted, otherwise it would crash with `django.core.exceptions.ValidationError: {'password': ['This field cannot be blank.']}`. Should this be documented in the patch notes?

**Comment by jobec** [comment]
Could you make a second issue for the password issue? That way we don't mix 2 problems.

As for the selective SSO part, there are 2 options:
1) Manually add the `&prompt=login` to the URL in a Django template like this: `<a class="nav-link" href="{% url 'django_auth_adfs:login' %}&prompt=login">Login</a>`
2) Add a new reverse URL named `django_auth_adfs:login_prompt` or something similar, add a view and a extra URL build function to `django_auth_adfs.config.ProviderConfig`. And alaso add it to the demo project as an extra URL.

PRs are welcome 😉

**Comment by JonasKs** [comment]
Updated and created a new ticket, see #65 .

Solution 1 seems fine to me and solved the issue, but if you'd like to have it documented then maybe solution 2 would be better?

**Comment by JonasKs** [comment]
Tried *solution 1*, it works as expected in Firefox but does not work in Chrome. It also only work once, so if one wish to change back to the SSO user, the browser has to be completely shut down. 
I guess the solution must have implemented the `django_auth_adfs:logout` and logic around it.

**Comment by jobec** [comment]
You're right... It's not going to work because the link is to a page on the same server that then redirects to ADFS. It's even strange that it worked on Firefox.

So solution 2 is the only way forward.

**Comment by JonasKs** [comment]
I don't have time to look into this before the weekend, but if no one else provides a PR before then I'll see what I can do. Thanks!

**Comment by jobec** [comment]
Added.
See the bottom of the installation documentation how to use it.

**Comment by jobec** [comment]
Please test it and if it works fine, I'll release it as v1.1.2

**Comment by JonasKs** [comment]
Can confirm it works as expected in Firefox and Chrome. ⭐️
